ga4gh/frontend.py

```diff
@@ -228,6 +274,9 @@ def configure(configFile=None, baseConfig="ProductionConfig", extraConfig={}):
     theBackend.setDefaultPageSize(app.config["DEFAULT_PAGE_SIZE"])
     theBackend.setMaxResponseLength(app.config["MAX_RESPONSE_LENGTH"])
     app.backend = theBackend
+    app.secret_key = os.urandom(SECRET_KEY_LENGTH)
```

Comment:
This is for the secure cookie, yeah?  Similar argument to the OIDC dynamic registration above.  It's fine if you run this server in Python's built-in single-threaded WSGI server, but as soon as you go to processes > 1 or worse spanning multiple machines, you'll have a bad time.

I handled this in the simplebeacon by [reading from a file](https://github.com/Genecloud/simplebeacon/blob/master/beacon_rp.py#L22), which can be shared among all running processes/machines.  Don't forget to add the filename of this key [to the .gitignore](https://github.com/Genecloud/simplebeacon/blob/master/.gitignore#L5).